src/routes/NotFound/NotFoundView.js

```diff
@@ -1,38 +1,64 @@
 // @flow
 
-import React from 'react'
+import React, { PureComponent } from 'react'
 
 import { JThumbnail, JFlatButton } from 'components/base'
 
 type Props = {|
   goToHome: () => void,
 |}
 
-const NotFoundView = ({ goToHome }: Props) => (
-  <div className='not-found-view'>
-    <div className='content'>
-      <JThumbnail
-        color='white'
-        iconSize='xlarge'
-        image='auth-question'
-        title='404. Page Not Found.'
-        description={[
-          'The page you\'re looking for can\'t be found.',
-          'Check the URL and try again.',
-        ]}
-      />
-      <div className='actions'>
-        <div className='back'>
-          <JFlatButton
-            onClick={goToHome}
+type StateProps = {|
+  isHovered: boolean,
+|}
+
+class NotFoundView extends PureComponent<Props, StateProps> {
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      isHovered: false,
+    }
+  }
+
+  onHover = () => this.setState({ isHovered: !this.state.isHovered })
```

Comment:
😭

Reply:
let's fix this - true on hover and false on leave